Retention article update (#5291)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/retention-policies.md

## How Teams retention policies work

Teams chats are stored in a hidden folder (Teamschat) in the mailbox of each user in the chat, and Teams channel messages are stored in a hidden folder(Teamschat) in the group mailbox for a team. Teams uses an Azure-powered chat service that also stores this data, and by default this service stores the data forever. With a Teams retention policy, when you delete data, the data is permanently deleted from both the Exchange mailboxes and the underlying chat service.

When you apply a **retention-hold** policy to Teams chats or channel messages, here's what happens:

> - If a chat or channel message is edited or deleted by a user during the retention-hold period, the message is copied (if it was edited) or moved (if it was deleted) to the SubstrateHolds folder and stored there until the retention period expires. If the policy is configured to delete data when the retention period expires, messages are permanently deleted on the day the retention period expires.

Reviewer: ```suggestion
- If a chat or channel message is edited or deleted by a user during the **retention-hold** period, the message is copied (if it was edited) or moved (if it was deleted) to the SubstrateHolds folder and stored there until the retention period expires. If the policy is configured to delete data when the retention period expires, messages are permanently deleted on the day the retention period expires.
```

- If a chat or channel message isn't deleted by a user during the **retention-hold** period, the message is moved to the SubstrateHolds folder within one day after the retention period expires. If the policy is configured to delete data when the retention period expires, the message is permanently deleted one day after it's moved to the folder.

When you apply a **retention-delete** policy to Teams chats and channel messages, here's what happens:

> - When a chat or channel message expires i.e. age of message is more than allowed by **retention-delete** policy, a back-end service, identifies expired messages and starts deleting them in the backend storage (user or group mailbox). 

Reviewer: ```suggestion
- When a chat or channel message expires, i.e. the age of the message is more than what is allowed by the **retention-delete** policy, a back-end service identifies expired messages and starts deleting them in the back-end storage (user or group mailbox). 
```

> - Once a message is deleted in back-end storage, a process is triggered to delete the same message in the Azure-powered chat service and user’s Teams app. For the messages to be deleted in Teams app, the app needs to be connected to internet and to be in idle state (no user activity), so that the deletion process would not interfere in user experience. Since a user might have multiple devices, which might be in different states, retention deletes would not sync up with those devices at exactly same time.

Reviewer: ```suggestion
- Once a message is deleted in the back-end storage, a process is triggered to delete the same message in the Azure-powered chat service and the user’s Teams app. For the messages to be deleted in the Teams app, the app needs to be connected to the internet and to be in idle state (no user activity), so that the deletion process would not interfere with the user experience. Since a user might have multiple devices, which might be in different states, retention deletes would not sync up with those devices at the exact same time.
```

> - Once the deletion of messages in backend storage is complete, those messages will stop showing up in compliance search reports such as eDiscovery.

Reviewer: ```suggestion
- Once the deletion of messages in back-end storage is complete, those messages will stop showing up in compliance search reports such as eDiscovery.
```

- Teams doesn't support advanced retention settings, such as the ability to apply a policy to content that contains keywords or sensitive information. Currently, retention policies in Teams apply to all chat and/or channel message content.

> - A Teams retention policy will trigger a process to delete chat and channel messages when those messages expire (based on message creation date). However, depending on service load, it may take up-to seven days to permanently delete these messages from backend storage and Teams app. Also, these messages will be searchable with compliance tools (eDiscovery, end user search) till they are permanently deleted from backend storage.

Reviewer: ```suggestion
- A Teams retention policy will trigger a process to delete chat and channel messages when those messages expire (based on message creation date). However, depending on service load, it may take up to seven days to permanently delete these messages from the back-end storage and Teams app. Also, these messages will be searchable with compliance tools (eDiscovery, end user search) until they are permanently deleted from the back-end storage.
```
